src/TraceParserGen/TraceParserGen.cs

```diff
@@ -1327,6 +1304,10 @@ private class StructInfo : FieldInfo
     /// </summary>
     private SortedDictionary<string, List<Event>> m_eventsByName;
 
-    private static Dictionary<string, string> reservedKeywords = new Dictionary<string, string> { { "object", "Object" }, { "new", "New" }, { "protected", "Protected" } };
+    /// <summary>
+    ///  Used in the generation of C#-valid identifiers
+    /// </summary>
+    static private Microsoft.CSharp.CSharpCodeProvider CS_provider = new Microsoft.CSharp.CSharpCodeProvider();
```

Comment:
❓ Will this map `object` to `Object` or to `@object`?

Reply:
It is enough to use HashSet here and just add '@' character in front of reserved name.

Reply:
In fact it maps `object` to `_object`